File: 프로그래머스/lv0/120912. 7의 개수/7의 개수.cpp
#include <vector>
#include <string>
#include <algorithm>
#include <sstream>
#include <iterator>

using namespace std;
int solution(vector<int> array)
{
    int count = 0;
    stringstream result;
    copy(array.begin(), array.end(), ostream_iterator<int>(result, ""));
    string n = result.str();

    for (int i = 0; i < n.size(); i++)
    {
        if (n[i] == '7')
            count++;
    }
    

    return count;
}

File: 프로그래머스/0/120912. 7의 개수/7의 개수.java
class Solution {
    public int solution(int[] array) {
        int count = 0;
        for (int i : array) {
            while (i != 0) {
                int remainder = i % 10;
                if (remainder == 7) count++;
                i /= 10;
            }
        }
        return count;
    }
}